client.go

```diff
@@ -67,14 +65,20 @@ func (c *Client) sendToStatsd(in chan *logMetrics) {
 	}
 }
 
-func (c *Client) sendRouterMsg(data *logMetrics) {
-
-	tags := *data.tags
-	for _, mk := range routerMetricsKeys {
-		if v, ok := data.metrics[mk]; ok {
+func (c *Client) extractTags(tags []string, permittedTags []string, metrics map[string]logValue) []string {
+	for _, mk := range permittedTags {
```

Comment:
No, not really. I've just factored the existing code out of `sendRouterMsg` and `sendMetricsMsg`, which appends to the tags. The only behaviour changed here is possibly excluding some of the tags.